function! health#broken#check()
  throw 'caused an error'
endfunction
